src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -36,8 +36,10 @@
 
 {% block form_label -%}
   {% spaceless %}
-    {% if label is same as(false) %}
+    {% if label is same as(null) %}
```

Comment:
is there a reason not to use `is not defined` here ?

Reply:
Yes label is always defined because it's a default option however it can contain a string, false or null
The idea is to play with the two different "falsy" values to have a different behaviour:
- null => we have an empty label column (to keep the "indentation" in the form fields comapred to other inputs which have labels associated)
- false => we have absolutely **no** label, not even an empty column, this is to remove this empty space that is not wanted for large fields (like multilang wysiwyg for example)